composer.json

```diff
@@ -44,7 +44,7 @@
   "config": {
     "bin-dir": "bin",
     "platform": {
-      "php": "8.0"
+      "php": "8.2"
```

Comment:
@tomglvng we have a suggested package down

    "php-http/guzzle6-adapter": "In order to use Guzzle v6 as the HTTP client"

Maybe we can update it as we are now on guzzle7 ?